src/main/java/uk/gov/hmcts/reform/sscs/mappers/HearingPriorityType.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.sscs.mappers;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum HearingPriorityType {
+    HIGH("high", "High Priority"),
+    NORMAL("normal", "Normal Priority");
+
+    private final String type;
+    private final String description;
+
+    public String getType() {
+        return this.type;
+    }
+
+    @Override
+    public String toString() {
+        return this.type;
+    }
+}
```

Comment:
Can you delete this class

Reply:
Address this for MVP

Reply:
Deleted. One usage, in a test. replaced with a literal equivalent, "high", for now.